students/juyeob/westagram/users/views.py

```diff
@@ -57,16 +57,25 @@ def post(self, request):
             email    = data['email']
             password = data['password']
 
+            user = User.objects.get(email = email)
+
             if not User.objects.filter(email = email).exists():
                 return JsonResponse({"message": "INVALID_USER"}, status=401)         
 
-            if not User.objects.filter(password = password).exists():    
-                return JsonResponse({"message": "INVALID_USER"}, status=401)
+            if not bcrypt.checkpw(password.encode('utf-8'), user.password.encode('utf-8')):
+                    return JsonResponse({"message": "INVALID_PASSWORD"}, status=401)
```

Comment:
로그인을 할 때는 아이디가 틀렸는지, 비밀번호가 틀렸는지 모르게 하는 것이 더 좋습니다.

```suggestion
                    return JsonResponse({"message": "INVALID_USER"}, status=401)
```